src/main/java/com/google/sps/utils/UserUtils.java

```diff
@@ -62,4 +66,54 @@ public Entity getEntityFromDatastore(String entityName, String entityPropertyTit
     Entity entity = resultsList.get(0);
     return entity;
   }
+
+  /**
+  * Adds a game id to a user's list of games
+  */
+  public static boolean addGameToUser(Entity userEntity, DatastoreService datastore, String gameId) {
+ 
+    if(gameId == ""){
+        log.severe("no gameId");
+        return false;
+    }
+
+    if(userEntity == null || datastore == null){
+        log.severe("null datastore or user entity");
```

Comment:
This isn't a super helpful logging statement. If I read this, I'll still have questions (specifically, "well, which one is it?"). This can be improved by having one if statement per check and logging at each step.

Reply:
I split them up and made them more specific